ml-agents-envs/mlagents/envs/environment.py

```diff
@@ -607,7 +607,7 @@ def _flatten(cls, arr) -> List[float]:
         arr = [float(x) for x in arr]
         return arr
 
-    def _get_state(self, output: UnityRLOutput) -> (AllBrainInfo, bool):
```

Comment:
Unfortunately that's not valid type annotation syntax, you have to use `Tuple[]`